webpack.config.prod.js

```diff
@@ -0,0 +1,25 @@
+const commonConfig = require('./webpack.config.common');
+const webpackMerge = require('webpack-merge');
+const webpack = require('webpack');
+const WebpackMd5Hash = require('webpack-md5-hash');
+
+module.exports = webpackMerge(commonConfig, {
+
+  output: {
+    filename: '[name].bundle.js'
+  },
+
+  plugins: [
+    new webpack.HotModuleReplacementPlugin(),
```

Comment:
Hey @Peter-holyoke , some thoughts here:
1. HMR should not be used in production, it's a development feature only.
1. We don't need to define `output` as a result
1. I forgot, if we use the `-p` flag, we get [**UglifyJS** for free](https://webpack.js.org/guides/production/)!
> And, webpack -p will automatically invoke both those flags and thus the plugins to be included.
4. Also, I think we should [add the **DefinePlugin**](https://webpack.js.org/guides/production/#specify-the-environment) like so:
> 
```
new webpack.DefinePlugin({
  'process.env': {
    'NODE_ENV': JSON.stringify('production')
  }
})
```
for this win
> If you're using a library like react, you should actually see a significant drop in bundle size after adding this plugin.